test/unit/app/controllers/swaps-test.js

```diff
@@ -79,14 +79,19 @@ describe('Swaps Controller', function () {
       slippagePercentage: '0.03',
       feeRecipient: '0xbd9420A98a7Bd6B89765e5715e169481602D9c3d',
       gasPrice: '21',
+    }
+    const qsPrice = createQueryString(baseQueryObj)
+    const qsQuote = createQueryString({
+      affiliateAddress: '0xbd9420A98a7Bd6B89765e5715e169481602D9c3d',
```

Comment:
Could we add a comment right here about what the purpose of this address and what it means for users?

If I understand correctly, this is address is owned by Brave, and we're using it to tag the transactions that are done using the Brave wallet? In other words, it's making possible to search for transactions from Brave users, but it's not exposing any more information about individual users since the transactions are already public?

Reply:
This a test file, but I added an explanation in [the right place](https://github.com/onyb/ethereum-remote-client/blob/1b05e90ac3e5b8de465a038b8c23c8384b9060c1/app/scripts/controllers/swap.js#L28-L33).

> If I understand correctly, this is address is owned by Brave, and we're using it to tag the transactions that are done using the Brave wallet? In other words, it's making possible to search for transactions from Brave users, but it's not exposing any more information about individual users since the transactions are already public?

You're completely right on all counts. Basically, the address used here is the same one we use for collecting swap fees since it uniquely identifies Brave. This is only for tagging transactions so we can see some aggregated metrics on [0xTracker](https://0xTracker.com). This is already public information available on the blockchain explorers.